ci/cloudbuild/builds/checkers.sh

```diff
@@ -131,14 +131,18 @@ time {
     xargs -P "$(nproc)" -n 50 -0 bash -c "sed_edit ${expressions[*]} \"\$0\" \"\$@\""
 }
 
-# Apply transformations to fix whitespace formatting in files not handled by
-# clang-format(1) above.  For now we simply remove trailing blanks.  Note that
-# we do not expand TABs (they currently only appear in Makefiles and Makefile
-# snippets).
+# Applies the following fixes in text files that do not say "DO NOT EDIT":
+#   - Removes trailing whitespace on lines
+#   - Removes trailing blank lines in files
 printf "%-30s" "Running whitespace fixes:" >&2
 time {
+  expressions=("-e" "'s/[[:blank:]]\+$//'")
+  # Trims trailing blank lines, courtesy of
+  # http://sed.sourceforge.net/sed1line.txt.
+  expressions+=("-e" ":a" "-e" "'/^\n*$/{\$d;N;ba'" "-e" "'}'")
```

Comment:
I of course do not understand if that works, or why, nor do I understand why anybody would want to understand why it works either.  If we ever have to fix this, I would remove the whole thing and start over in C++ or awk or python or something.

Reply:
Ack to all of that. I don't understand it either. If it breaks, we can just remove the line and go back to not caring (which I'm fine w/ even now).

Reply:
It boils down to ...
```
while (!eof) {
  read line into pattern space;
  while (pattern space matches /^\n*$/) {
    if (eof) {
      clear pattern space;
      break;
    }
    append line to pattern space;
  }
  print pattern space;
}
```
So, we're accumulating empty lines into the pattern space, and if we reach end-of-file before anything else we'll discard them.

The only thing I might suggest is coalescing it all into a single `-e` argument:
```
  expressions+=('-e' ':a;/^\n*$/{$d;N;ba;}')
```

Reply:
Done.